backend/data_import/pipeline/examples/article_annotation/example.json

```diff
@@ -1,16 +1,15 @@
 [
     {
-        "text": "Stół z powyłamywanymi nogami",
-        "cats": ["sadness", "negative"],
-        "entities": [[0, 3, "rzeczownik"], [7, 27, "narzędnik"]],
-        "relations": [],
+        "text": "Firma Biometryks LLC planuje zrewolucjonizować nieinwazyjną diagnostykę oraz monitorowanie pacjentów z chorobami przewlekłymi poprzez wykorzystanie w swoim sensorze matrycy grafenowej, która jest kluczowym elementem tego urządzenia - poinformował Uniwersytecki Ośrodek Transferu Technologii UW w przesłanym PAP komunikacie.",
+        "cats": [],
```

Comment:
I kept it but in form of empty list because I copy 1 example from the current exported data, so it's still there XD just the empty list. But oke I can change if it's needed XD

Reply:
Ah okie xD  its good either way tho

Reply:
Okie I just added the cats and entities example in the recent commit XD